app/posts/manage-teacher-training-applications/2021-02-01-issues-with-changing-an-offer.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Issues with changing an offer
+description: Various issues with changing the details of an offer as a user who belongs to an accredited body or a school direct
+date: 2021-02-01
+---
+
+The way changing an offer works as an accredited body is buggy, confusing and inconsistent with how it works as a training provider.
+
+This post explains some of the issues.
+
+## Changing the training provider if you’re an accredited body
+
+Scenario:
+
+> Given I belong to an accredited body
+> Who ratifies courses at multiple training providers
+> And I have the ‘make decisions’ permission
+> When I click ‘change provider’
+> I am taken to a page with just my accredited body as a pre-selected radio button
+
+Issues:
+
+- There shouldn’t be a ‘change’ link if the user cannot actually change the provider.
+- It says ‘provider’ on the offer panel and it says ‘training provider’ on the next screen, but it actually shows the accredited body.
+
+Questions:
+
+- When changing the provider should we give the user a list of all school direct partners? Probably.
+
+## Changing the course if you’re an accredited body
+
+Scenario:
+
+> Given I belong to an accredited body
+> Who ratifies courses at the training provider
+> And I have the ‘make decisions’ permission
+> When I click ‘change course’
+> I am taken to a page that lists all the courses that the training provider runs
+
+Issues:
+
+- The user should only see courses that they accredit.
+- The ‘change link’ appears because the selected training provider has multiple courses - it should only appear if the currently selected training provider runs multiple courses that the accredited body ratifies.
```

Comment:
```suggestion
- The ‘change’ link appears because the selected training provider has multiple courses - it should only appear if the currently selected training provider runs multiple courses that the accredited body ratifies.
```